.github/workflows/init.yml

```diff
@@ -6,6 +6,9 @@ on:
       collector-tag:
         description: The tag used when building collector
         value: ${{ jobs.common-variables.outputs.collector-tag }}
+      collector-qa-tag:
```

Comment:
I think this is currently unused (at least from this source) since it's set by the `integration-test-containers.yml` workflow 

We can probably use either source to setup the necessary qa tag for master vs. pull requests, but it's worth converging on a single source. I sort of think that `init.yml` is the right place for this kind of setting, but I'm not overly attached to that opinion so use whichever you think is best.

Reply:
I initially meant for this to be a backup in case the `integration-test-containers.yml` workflow was skipped, but I guess that changed at some point and forgot to unify.

If I move it to be here I'll have to also add a duplicated step checking if `QA_TAG` was modified, but that should be fine (I can actually check it here and set another output for the other step now that I think about it).